lib/recipes-data/src/lib/config.ts

```diff
@@ -29,6 +29,6 @@ export const TelemetryXAR = process.env["TELEMETRY_XAR"]
 export const TelemetryTopic = process.env["TELEMETRY_TOPIC"]
 
 //Used by content transforms
-export const PreviewImageWidth = process.env["PREVIEW_IMAGE_WIDTH"] ? parseInt(process.env["PREVIEW_IMAGE_WIDTH"]) : 300;
-export const FeaturedImageWidth = process.env["FEATURED_IMAGE_WIDTH"] ? parseInt(process.env["FEATURED_IMAGE_WIDTH"]) : 700;
+export const PreviewImageWidth = process.env["PREVIEW_IMAGE_WIDTH"] ? parseInt(process.env["PREVIEW_IMAGE_WIDTH"]) : 1600;
+export const FeaturedImageWidth = process.env["FEATURED_IMAGE_WIDTH"] ? parseInt(process.env["FEATURED_IMAGE_WIDTH"]) : 1600;
```

Comment:
Are there any lower-bounds for what could be set in the env vars? E.g. if we have an env var that says 300 we'd still select that over 1600?

Reply:
I'm tempted to say caveat emptor on the env vars, which would arrive from the stack – but we could add this if we thought it was a sensible safeguard.